src/generic/templates/audio.js

```diff
@@ -57,12 +59,27 @@ class Audio extends Template {
     const {
       title,
       url,
+      duration,
       action,
       quickReplies,
       delay,
       fallback
     } = this
 
+    if(duration){
```

Comment:
Why are we duplicating return? Can't you compose one object conditionally and return it? And what is the problem behind having duration in the return object by default? Won't it be undefined and then omitted during serialization?